service/EmailService.cs

```diff
@@ -0,0 +1,32 @@
+﻿using service.Interfaces;
+using System.Net.Mail;
+using System.Net;
+
+namespace service
+{
+    public class EmailService : IEmailService
+    {
+        public void EnviarEmail(string emailDestinatario, string assunto, string corpo)
+        {
+
+            MailMessage mensagem = new MailMessage();
+
+            string emailRemetente = "email@gmail.com";
```

Comment:
Aqui deve-se utilizar aquele e-mail que estamos utilizando para contas do dnit, no final do projeto podemos trocar para um e-mail oficial do dnit. Mas por enquando utilize aquele pois se não a implementação vai falhar quando for testada pelos POs. Além disso, como tem e-mail e a senha (senha do app), isso deve ser guardado nas variáveis de ambiete. Não esqueça de fazer isso por segurança

Reply:
Atualizei para usar variáveis de ambiente e já configurei.

Reply:
Boooa